conduct/enforcement.md

```diff
@@ -0,0 +1,208 @@
+# Jupyter Code of Conduct - Enforcement Manual
+
+This is the enforcement manual followed by Jupyter's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're consistent
+and fair.
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee"), currently comprised of [*note:* still gathering volunteers, this will be filled in as they become available and prior to final upload into the rendered site]:
```

Comment:
@jasongrout I believe this was a note to distinguish adding things to the CoC here before uploading this content to the Jupyter website. Perhaps it would be good to mention: volunteers will be filled in on this markdown file as they become available. Alternatively, placeholder text for each member could be added. Leaving up to @fperez.